exercises/zebra-puzzle/src/main/kotlin/ZebraPuzzle.kt

```diff
@@ -0,0 +1,8 @@
+class ZebraPuzzle() {
+
+    var waterDrinker: String = TODO("Implement this class to complete the task")
+        private set
+
+    var zebraOwner: String = TODO("Implement this class to complete the task")
+        private set
```

Comment:
Here we share reference implementation details. But a student is not obliged to stick with our implementational approach. 

Can we declare two extra functions like
```kotlin
fun drinksWater(): String
fun ownsZebra(): String
```
and retarget test to  them? 

They will be return `waterDrinker` and `zebraOwner` respectively for the case of refreence implementation, and will let a student implement 'em as one desires?

Reply:
Is done!